test/e2e/e2e.go

```diff
@@ -143,6 +139,10 @@ var _ = ginkgo.BeforeEach(func() {
 	suiteCfg.EmitSpecProgress = true
 	// Randomize specs as well as suites
 	suiteCfg.RandomizeAllSpecs = true
+	// Disable skipped tests unless they are explicitly requested.
+	if len(suiteCfg.FocusStrings) == 0 && len(suiteCfg.SkipStrings) == 0 {
```

Comment:
commented in https://github.com/kubernetes/kubernetes/pull/109111/commits/64d29d99607b520ab3390a25406f8c940acaa6fe#r857104476